scripts/ts_indices/calculate_dowdy_ts_index.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#PBS -Pw85
+#PBS -qnormal
+#PBS -m ae
+#PBS -M kieran.ricardo@ga.gov.au
+#PBS -l walltime=0:05:00
+#PBS -lmem=128GB,ncpus=48,jobfs=4000MB
+#PBS -joe
+#PBS -l storage=scratch/w85+gdata/w85+gdata/rt52+gdata/dk92+gdata/v10
+
+module purge
+module use /g/data/v10/public/modules/modulefiles
```

Comment:
Need to be sure we can use modules that are readily available. There's a bunch that are made available to all NCI users (but still need project membership). Not sure how happy DEA would be with everyone wanting to join just to use their modules.